workers/tasks/export/notes.ts

```diff
@@ -30,51 +34,22 @@ const enrichNoteWithStructuredResult = (
 	return note;
 };
 
-const getPlainUnstructuredTextForNote = (note: Note): null | string => {
-	if (note.unstructuredValue) {
-		return callPandoc(note.unstructuredValue, 'html', 'plain').trim();
-	}
-	return null;
-};
-
 const getCslJsonForNote = (
 	note: Note,
 	hash: string,
 	renderedStructuredValues: RenderedStructuredValues,
-) => {
+): [boolean, Record<string, any>] => {
```

Comment:
Could we maybe type this tuple or name its elements? I guess it's kind of obvious by looking at the function implementation but it wouldn't be so clear otherwise.